locale/tr_TR/editor.xml

```diff
@@ -192,26 +217,24 @@
 	<message key="editor.article.hideTocAuthorHide">Ele</message>
 	<message key="editor.article.hideTocAuthorShow">Göster</message>
 	<message key="editor.article.cover">Kapak</message>
+    	<message key="editor.article.coverImage">Kapak resmi</message>
 	<message key="editor.article.showCoverPage">Aşağıdaki öğeler ile bu makale için bir kapak oluştur.</message>
 	<message key="editor.article.coverImageDisplay">Görüntüle</message>
 	<message key="editor.article.hideCoverPageToc">İçindekiler sayfasında küçük kapak görseli görüntüleme.</message>
 	<message key="editor.article.hideCoverPageAbstract">Makale abstraktını görüntülerken kapak görseli görüntüleme.</message>
-	<message key="editor.article.coverImage">Kapak resmi</message>
+	
 	<message key="editor.article.coverImageInstructions">(Kabul edilebilir formatlar: .gif, .jpg veya .png )</message>
 	<message key="editor.article.uploaded">Yüklendi</message>
 	<message key="editor.article.removeCoverPage">Kapak görseli kaldırılsın mı?</message>
+   	<message key="grid.action.removeArticle">Makale Sayıdan Silinsin</message>
+    	<message key="editor.article.remove">Sil</message>
+    	<message key="editor.article.remove.confirm">Makaleyi bu sayıdan silmek istediğinizden emin misiniz ? Bu makale bir başka sayı içinde yayınlanabilir.</message>
 	<message key="editor.issues.invalidAccessDate">Giriş tarihi geçerli değil.</message>
+   	<message key="editor.issues.editIssueGalley">Sayı Dizgisini Düzenle</message>
+   	<message key="editor.submission.production.productionReadyFilesDescription"><![CDATA[Mizanpaj editörü bu dosyaları her dizgi için hazırlar ve sağlama <span class="pkp_help_title">Sağlama</span> için yükler.]]></message>
+    	<message key="editor.submission.selectGalleyFiles">Dizgi dosyalarını seç</message>
+    	<message key="editor.submission.editorial.finalDraftDescription"><![CDATA[Bu listedeki nihai taslak dosyalar altında aşağıda yüklendi copyedited sürümleri, bir sayfa düzenleyici tarafından dönüştürülür.<span class="pkp_help_title">Copyediting</span>.]]></message>
```

Comment:
Did you leave this in English on purpose? This is just the title of that chapter in the documentation, you could translate this, too...